tests/e2e/Services/Functions/FunctionsCustomServerTest.php

```diff
@@ -1984,4 +1985,25 @@ public function testFunctionLogging()
 
         $this->cleanupFunction($functionId);
     }
+
+    /**
+     * @depends testCreateFunction
+     */
+    public function testUniqueSubdomain(array $data): void
```

Comment:
We can keep this logic in `testCreateFunction`, we can make section thefer for:
```
        /**
         * Test for FAILURE
         */
```

Its pattern we already use in tests